cfme/tests/openstack/cloud/test_flavors.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for Openstack cloud Flavors"""
+
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.usefixtures("setup_provider"),
+    pytest.mark.provider([OpenStackProvider], scope='function')
+]
+
+
+DISK_SIZE = 1
+RAM = 64
+VCPUS = 1
+SWAP_SIZE = 1
+RXTX = 1
+ZERO_DISK_SIZE = 0
+
+
+@pytest.fixture(scope='function')
+def flavor(appliance, provider):
+    collection = appliance.collections.cloud_flavors
```

Comment:
please put this on one line since the args are split anyway, `flavor = appliance.collections.cloud_flavors.create(` will fit on 100 chars.